src/server.c

```diff
@@ -6612,6 +6613,7 @@ struct serverTest {
     {"zipmap", zipmapTest},
     {"dict", dictTest},
     {"listpack", listpackTest},
+    {"rax", raxTest},
```

Comment:
This is for our old unit tests. It isn't needed for the new unit tests under src/unit/. (We should port all the old ones to the new way, some day.)

Reply:
Makes sense, I've removed this function now so that individual rax tests are plugged in the new framework, under src/unit/test_rax.c.

I've also mapped some of the tests to the existing flags (--large-memory|--accurate|--single).